app/brave_generated_resources.grd

```diff
@@ -730,6 +730,12 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
       <message name="IDS_SETTINGS_IPFS_AUTO_FALLBACK_TO_GATEWAY_DESC" desc="The description for the switch of automatically fallback to IPFS public gateway">
         Automatically fallback to IPFS public gateway when the local node cannot be accessed.
       </message>
+      <message name="IDS_SETTINGS_IPFS_AUTO_REDIRECT_GATEWAY_LABEL" desc="The text label for redirecting IPFS resources to the configured gateway">
+        Redirect IPFS resources gateway URLs
+      </message>
+      <message name="IDS_SETTINGS_IPFS_AUTO_REDIRECT_GATEWAY_DESC" desc="The description for for redirecting IPFS resources to the configured gateway">
```

Comment:
remove the extra for in the desc.

Reply:
Fixed via interactive rebase of the commit b763d0b86bd0c07dccf336fc0e4f6519c959a6a7